### PR TITLE
Add links to ItemsRepeater page

### PR DESCRIPTION
**Pull request by anawishnoff**
## Description
Adding links to UniformGridLayout and StackLayout APIs as these are mentioned in the samples.

## Motivation and Context

Addresses issue #369.

## How Has This Been Tested?

Tested to make sure links work.
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

File: XamlControlsGallery/ControlPages/ItemsRepeaterPage.xaml [hunk 1]
```diff
@@ -255,7 +255,8 @@ the following attributes: Height, MaxHeight, Length, MaxLength, Diameter, and Ma
                                         Click="Animated_GotItem" 
                                         GotFocus="Animated_GotItem" 
                                         HorizontalAlignment="Stretch"
-                                        Loaded="GetButtonSize"/>
+                                        Loaded="GetButtonSize"
+                                        Foreground="{StaticResource SystemBaseHighColor}"/>
```
Comment (chingucoding) [inline XamlControlsGallery/ControlPages/ItemsRepeaterPage.xaml]: Removing the foreground did not seem to make a difference, was there a specific reason why this was added? 😅

Comment (stmoy) [reply XamlControlsGallery/ControlPages/ItemsRepeaterPage.xaml]: +1 - this seems unrelated to the issue. Maybe this is leftover from a different fix? In either case, please remove.

File: XamlControlsGallery/DataModel/ControlInfoData.json [hunk 2]
```diff
@@ -1304,6 +1304,14 @@
             {
               "Title": "Guidelines",
               "Uri": "https://docs.microsoft.com/windows/uwp/design/controls-and-patterns/items-repeater"
+            },
+            {
+              "Title": "StackLayout - API",
+              "Uri": "https://docs.microsoft.com/en-us/uwp/api/microsoft.ui.xaml.controls.stacklayout?view=winui-2.3"
```
Comment (stmoy) [inline XamlControlsGallery/DataModel/ControlInfoData.json]:
```suggestion
              "Uri": "https://docs.microsoft.com/uwp/api/microsoft.ui.xaml.controls.stacklayout?view=winui-2.3"
```

File: XamlControlsGallery/DataModel/ControlInfoData.json [hunk 3]
```diff
@@ -1304,6 +1304,14 @@
...
+            },
+            {
+              "Title": "UniformGridLayout - API",
+              "Uri": "https://docs.microsoft.com/en-us/uwp/api/microsoft.ui.xaml.controls.uniformgridlayout?view=winui-2.3"
```
Comment (stmoy) [inline XamlControlsGallery/DataModel/ControlInfoData.json]:
```suggestion
              "Uri": "https://docs.microsoft.com/uwp/api/microsoft.ui.xaml.controls.uniformgridlayout?view=winui-2.3"
```

**Pull request merged**